deploy/clientcomm.tf

```diff
@@ -226,6 +226,20 @@ resource "aws_route_table_association" "clientcomm" {
   route_table_id = "${aws_route_table.clientcomm.id}"
 }
 
+// TODO: provision the DNS zone here instead of looking it up?
+data "aws_route53_zone" "clientcomm" {
+  // transform 'https://multnomah.clientcomm.org' -> 'clientcomm.org.'
+  name = "${replace(replace(var.deploy_base_url, "/https:\\/\\//", ""), "/^[^\\.]+\\./", "")}."
```

Comment:
Should maybe use a uri parsing library like [urijs](https://github.com/medialize/URI.js) for this kind of thing, to minimize possibility of error.